.eslintrc.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     },
     'extends': 'eslint:recommended',
     'parserOptions': {
-        'ecmaVersion': 12
+        'ecmaVersion': 2020
```

Comment:
tried to fix a formatting error but i just ended up disabling it for that document 😂